torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2413,10 +2413,6 @@ def sample_inputs_gather(op_info, device, dtype, requires_grad, **kwargs):
         make_arg((M, S)),
         1,
         gather_variable((M, S // 2), 0, S, True, device=device))
-    yield SampleInput(
-        make_arg(),
-        0,
-        torch.tensor([0], dtype=torch.int64, device=device))
```

Comment:
Repeated sample.

Reply:
@lezcano one of these is `torch.tensor([0])` (i.e. 1d) and the other is `torch.tensor(0)` (i.e. 0d)

Reply:
Ah, right, sorry for that. Reverting at https://github.com/pytorch/pytorch/pull/107776